hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -103,7 +103,7 @@ <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackatho
                     {% elif review.status == "Waitlisted" %}
                         <h4 class="formH2">You've been waitlisted for {{ hackathon_name }}</h4>
                         <p>The {{ hackathon_name }} team has reviewed your application, and have decided not to grant you a guaranteed spot
-                            at {{ hackathon_name }} and to place you in our waitlist. On {{ waitlisted_acceptance_start_time.strftime("%b %d %Y, at %I:%M%p") }}, we will begin allowing
+                            at {{ hackathon_name }} and to place you in our waitlist. On {{ waitlisted_acceptance_start_time.strftime("%B %-d, %Y, at %I:%M%p") }}, we will begin allowing
```

Comment:
Not sure why I didn't attach my comment here in the last review. Change to `"%B %-d, %Y, at %-I:%M %p"`